### ISSUE TITLE
The ICLabel IC scalp maps plot should also show IC numbers (as 'IC17' etc.) above-left each cartoon head.

### ISSUE THREAD

**Issue by dungscout96** [sccn/NEMAR-pipeline#45]

**Comment by dungscout96** [comment]
remove '-noui' in print(gcf) or iterating through subplots adding title
```
for iPlot = 1:32
    subplot(6.6.iPlot)
    title('test')
end
```

**Comment by dungscout96** [comment]
fixed in  431b471
